python-package/mlbox/encoding/categorical_encoder.py

```diff
@@ -9,7 +9,7 @@
 import os
 
 # Set the keras backend if not set, default is theano
-if not "KERAS_BACKEND" in os.environ:
+if "KERAS_BACKEND" not in os.environ:
     os.environ["KERAS_BACKEND"] = "theano"
 
 from keras.layers.core import Dense, Reshape, Dropout
```

Comment:
@AxeldeRomblay could you explain the need to put the keras backend here?
Initially ` $HOME/.keras/keras.json` sets the backend to `tensorflow`.